railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
@lanks sorry for not being able to explain well. I thought my logic above was wrong in some cases, so I've updated my comment above. Correct me if I'm still missing :+1: